src/external/utils/druidExpressionBuilder.ts

```diff
@@ -143,6 +143,10 @@ export class DruidExpressionBuilder {
           case 'boolean':
             return String(Number(literalValue));
 
+          case 'object':
+            if ( literalValue instanceof Date) return DruidExpressionBuilder.escapeLiteral(literalValue);
```

Comment:
```suggestion
            if (literalValue instanceof Date) return DruidExpressionBuilder.escapeLiteral(literalValue);
```
Plywood doesn't have Prettier I guess?